packages/core/test/unit/di/buildMainInjector.spec.ts

```diff
@@ -98,6 +98,7 @@ describe(buildMainInjector.name, () => {
     const expected: MutatorDescriptor = {
       name: 'javascript',
       plugins: null,
+      appendPlugins: null,
```

Comment:
Plugins of the mutator descriptor is a list of _babel plugins for parsing source files_, it doesn't have anything to do with `appendPlugins` 😉